java/integTest/src/test/java/glide/standalone/TransactionTests.java

```diff
@@ -121,4 +122,18 @@ public void lastsave() {
         var response = client.exec(new Transaction().lastsave()).get();
         assertTrue(Instant.ofEpochSecond((long) response[0]).isAfter(yesterday));
     }
+
+    @Test
+    @SneakyThrows
+    public void objectFreq() {
+        String objectFreqKey = "key";
+        Transaction transaction = new Transaction();
+        transaction.configSet(Map.of("maxmemory-policy", "allkeys-lfu"));
```

Comment:
should we revert this configuration after the test so that it doesn't affect other tests?